Dockerfile

```diff
@@ -6,9 +6,9 @@ FROM openjdk:8u121-jdk-alpine as BUILD
 
 RUN apk add --no-cache curl tar bash
 
-ARG MAVEN_VERSION=3.5.2
+ARG MAVEN_VERSION=3.3.9
```

Comment:
Why not use version 3.5.3 instead ?

Reply:
Never mind, I could not find why version 3.5.2 is not available anymore, it seems that 3.3.9 is here to stay and your PR works and fixes the issue so I will merge it. Thank you!

Reply:
I tried the 3.5.3 but surprisingly it didn't work. Below there is the output of  docker trying to build eclair against maven 3.5.3 and it fails with "FileNotFoundException" when assembling the artifact for the submodule "eclair-node". It looks like the capsule plugin tries to get the artifact for eclair-core from "../eclair-core/target" instead of "~/.m2/repositories" as it should, i did not dig too much into it as this issue come up only with maven 3.5.3 (and officially eclair uses 3.3.x), perhaps is something to keep in mind for the future.

>Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/aether/aether-api/1.1.0/aether-api-1.1.0.jar (136 kB at 104 kB/s)
[INFO] [CapsuleMavenPlugin] [Capsule Version]: 1.0.3
[INFO] [CapsuleMavenPlugin] [Output Directory]: /usr/src/eclair-node/target
[INFO] [CapsuleMavenPlugin] [Build Info]: includeApp includeAppDep includePluginDep includeCompileDep includeRuntimeDep includeTransitiveDep
[INFO] [CapsuleMavenPlugin] [Capsule Jar File]: eclair-node-0.2-SNAPSHOT-${git.commit.id.abbrev}.jar
[INFO] [CapsuleMavenPlugin] 	[Manifest]:
[INFO] [CapsuleMavenPlugin] 		Application-Name: eclair-node-0.2-SNAPSHOT-${git.commit.id.abbrev}
[INFO] [CapsuleMavenPlugin] 		Embedded-Artifacts: fr.acinq.eclair:eclair-node_2.11:0.2-SNAPSHOT com.google.j2objc:j2objc-annotations:1.1 com.fasterxml.jackson.core:jackson-core:2.8.4 org.scala-lang:scala-library:2.11.11 org.codehaus.mojo:animal-sniffer-annotations:1.14 com.chuusai:shapeless_2.11:2.3.2 com.izettle:metrics-influxdb:1.1.8 org.json4s:json4s-ast_2.11:3.5.3 org.scala-lang.modules:scala-parser-combinators_2.11:1.0.4 org.scala-lang.modules:scala-java8-compat_2.11:0.7.0 org.json4s:json4s-scalap_2.11:3.5.3 org.zeromq:jeromq:0.4.0 commons-codec:commons-codec:1.9 com.typesafe.akka:akka-parsing_2.11:10.0.11 org.json4s:json4s-core_2.11:3.5.3 fr.acinq:bitcoin-lib_2.11:0.9.17 com.google.guava:guava:24.0-android org.codehaus.janino:janino:3.0.7 com.thoughtworks.paranamer:paranamer:2.8 io.dropwizard.metrics:metrics-healthchecks:4.0.1 org.json4s:json4s-jackson_2.11:3.5.3 com.fasterxml.jackson.core:jackson-annotations:2.8.0 com.google.code.findbugs:jsr305:3.0.2 org.checkerframework:checker-compat-qual:2.0.0 org.xerial:sqlite-jdbc:3.21.0.1 nl.grons:metrics4-akka_a24_2.11:4.0.1 ch.qos.logback:logback-classic:1.2.3 io.dropwizard.metrics:metrics-core:4.0.1 org.clapper:grizzled-slf4j_2.11:1.3.1 nl.grons:metrics4-scala_2.11:4.0.1 jgraph:jgraph:5.13.0.0 com.typesafe.akka:akka-http-core_2.11:10.0.11 com.fasterxml.jackson.core:jackson-databind:2.8.4 org.reactivestreams:reactive-streams:1.0.0 com.google.errorprone:error_prone_annotations:2.1.3 org.codehaus.janino:commons-compiler:3.0.7 org.scodec:scodec-bits_2.11:1.1.2 com.madgag.spongycastle:core:1.58.0.0 de.heikoseeberger:akka-http-json4s_2.11:1.19.0 org.scala-lang.modules:scala-xml_2.11:1.0.2 io.dropwizard.metrics:metrics-jvm:4.0.1 com.typesafe.akka:akka-slf4j_2.11:2.4.20 org.typelevel:macro-compat_2.11:1.1.1 ch.qos.logback:logback-core:1.2.3 org.scodec:scodec-core_2.11:1.10.3 org.antlr:antlr4-runtime:4.5.3 com.typesafe.akka:akka-http_2.11:10.0.11 com.typesafe:config:1.3.0 org.jgrapht:jgrapht-core:1.0.1 org.slf4j:slf4j-api:1.7.25 com.typesafe:ssl-config-core_2.11:0.2.1 com.typesafe.akka:akka-actor_2.11:2.4.20 org.jgrapht:jgrapht-ext:1.0.1 com.typesafe.akka:akka-stream_2.11:2.4.20 fr.acinq.eclair:eclair-core_2.11:0.2-SNAPSHOT 
[INFO] [CapsuleMavenPlugin] 		Premain-Class: Capsule
[INFO] [CapsuleMavenPlugin] 		Application-Class: fr.acinq.eclair.Boot
[INFO] [CapsuleMavenPlugin] 		Build-Info: includeApp includeAppDep includePluginDep includeCompileDep includeRuntimeDep includeTransitiveDep
[INFO] [CapsuleMavenPlugin] 		Manifest-Version: 1.0
[INFO] [CapsuleMavenPlugin] 		Main-Class: Capsule
Downloading from sonatype snapshots: https://oss.sonatype.org/content/repositories/snapshots/co/paralleluniverse/capsule/1.0.3/capsule-1.0.3.jar
Downloading from central: https://repo.maven.apache.org/maven2/co/paralleluniverse/capsule/1.0.3/capsule-1.0.3.jar
Downloaded from central: https://repo.maven.apache.org/maven2/co/paralleluniverse/capsule/1.0.3/capsule-1.0.3.jar (77 kB at 486 kB/s)
[INFO] [CapsuleMavenPlugin] 	[App] App jar embedded (eclair-node_2.11-0.2-SNAPSHOT.jar)
[INFO] [CapsuleMavenPlugin] 	[Embedded-Dependency] io.dropwizard.metrics:metrics-jvm:4.0.1(compile)
[INFO] [CapsuleMavenPlugin] 	[Embedded-Dependency] com.typesafe.akka:akka-parsing_2.11:10.0.11(compile)
[INFO] [CapsuleMavenPlugin] 	[Embedded-Dependency] com.fasterxml.jackson.core:jackson-annotations:2.8.0(compile)
[INFO] [CapsuleMavenPlugin] 	[Embedded-Dependency] org.typelevel:macro-compat_2.11:1.1.1(compile)
[INFO] [CapsuleMavenPlugin] 	[Embedded-Dependency] org.json4s:json4s-ast_2.11:3.5.3(compile)
[INFO] [CapsuleMavenPlugin] 	[Embedded-Dependency] org.codehaus.mojo:animal-sniffer-annotations:1.14(compile)
[INFO] [CapsuleMavenPlugin] 	[Embedded-Dependency] com.typesafe:ssl-config-core_2.11:0.2.1(compile)
[INFO] [CapsuleMavenPlugin] 	[Embedded-Dependency] com.thoughtworks.paranamer:paranamer:2.8(compile)
[INFO] [CapsuleMavenPlugin] 	[Embedded-Dependency] com.typesafe.akka:akka-http-core_2.11:10.0.11(compile)
[INFO] [CapsuleMavenPlugin] 	[Embedded-Dependency] com.typesafe.akka:akka-stream_2.11:2.4.20(compile)
java.io.FileNotFoundException: /usr/src/eclair-core/target/eclair-core_2.11-0.2-SNAPSHOT.jar (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at com.github.chrisdchristo.capsule.CapsuleMojo.addDependencies(CapsuleMojo.java:561)
	at com.github.chrisdchristo.capsule.CapsuleMojo.build(CapsuleMojo.java:358)
	at com.github.chrisdchristo.capsule.CapsuleMojo.execute(CapsuleMojo.java:318)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:956)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:290)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:194)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] eclair_2.11 0.2-SNAPSHOT ........................... SUCCESS [01:51 min]
[INFO] eclair-core_2.11 ................................... SUCCESS [05:11 min]
[INFO] eclair-node 0.2-SNAPSHOT ........................... FAILURE [ 10.284 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 07:13 min
[INFO] Finished at: 2018-05-23T09:42:40Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.chrisdchristo:capsule-maven-plugin:1.5.1:build (default) on project eclair-node_2.11: /usr/src/eclair-core/target/eclair-core_2.11-0.2-SNAPSHOT.jar (No such file or directory) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :eclair-node_2.11
The command '/bin/sh -c mvn install -pl eclair-node -am clean' returned a non-zero code: 1